lib/models/rocket.py

```diff
@@ -17,71 +16,29 @@ class CoordinateSystemOrientation(str, Enum):
 
 
 class Parachute(BaseModel):
-    name: str = "Main"
-    cd_s: float = 10
-    sampling_rate: int = 105
-    lag: float = 1.5
-    trigger: Union[str, float] = "apogee"
-    noise: Tuple[float, float, float] = (0, 8.3, 0.5)
+    name: str
+    cd_s: float
+    sampling_rate: int
+    lag: float
+    trigger: Union[str, float]
+    noise: Tuple[float, float, float]
 
 
 class Rocket(BaseModel):
     # Required parameters
-    motor: Motor = Motor()
-    radius: float = 0.0632
-    mass: float = 16.235
-    motor_position: float = -1.255
-    center_of_mass_without_motor: int = 0
-    inertia: Tuple[float, float, float] = (6.321, 6.321, 0.0346)
-    power_off_drag: List[Tuple[float, float]] = [
-        (0.0, 0.0),
-        (0.1, 0.1),
-        (0.2, 0.2),
-    ]
-    power_on_drag: List[Tuple[float, float]] = [
-        (0.0, 0.0),
-        (0.1, 0.1),
-        (0.2, 0.2),
-    ]
+    motor: Motor
+    radius: float
+    mass: float
+    motor_position: float
+    center_of_mass_without_motor: int
+    inertia: Tuple[float, float, float]
```

Comment:
inertia can also be a tuple of 6 arguments

Reply:
patched in [eb3d79a](https://github.com/RocketPy-Team/Infinity-API/pull/34/commits/eb3d79a90f706f8fff161898ea85dc41b96f431e)